tests/python/shared/fixtures/init.py

```diff
@@ -218,6 +233,9 @@ def create_compose_files(container_name_files):
                 if service_name in ("cvat_server", "cvat_utils"):
                     service_env = service_config["environment"]
                     service_env["DJANGO_SETTINGS_MODULE"] = "cvat.settings.testing_rest"
+                if service_name in CODE_COVERED_CONTAINERS:
+                    service_env = service_config["environment"]
+                    service_env["COVERAGE_PROCESS_START"] = ".coveragerc"
```

Comment:
It's a little bit confusing to me that the `COVERAGE_PROCESS_START` variable contains a path to the coverage config tools. Maybe it's better to choose a more suitable name for it

Reply:
This variable is needed for coverage.py. We cannot rename it